docs/en/contributing-to-appium/appium-from-source.md

```diff
@@ -174,4 +156,4 @@ the commit from occurring.
 
 Once code is committed and a [pull request](https://help.github.com/articles/about-pull-requests/)
 is made to the correct Appium respository on [GitHub](https://github.com/), Appium build system
-will run all of the functional tests.
+will run all of the functional tests. 
```

Comment:
nit: trailing spaces

Reply:
we should adopt prettier, or something 😄

Reply:
I  have simply enabled trimming in my VSCode

Reply:
trailing spaces are significant in github-flavored markdown, so that might be bad.  even with an `.editorconfig` file -- which most editors/IDEs support -- you'd have to skip trimming markdown.  prettier is smart enough to know how to handle this.